### PR TITLE
f-compatibility@v0.1.0 Browser compatibility service

### PR DESCRIPTION
**Pull request by simonsandersje**
Create service scaffold.

### CODE REVIEW

File: packages/services/f-compatibility/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+
+v0.1.0
+------------------------------
+*October 14, 2021*
+
+### Added
+- Add your change under the following headings: `Added`/`Changed`/`Deprecated`/`Removed`/`Fixed`/`Security`
```
Comment (kevinrodrigues) [inline packages/services/f-compatibility/CHANGELOG.md]: Do we need to change this to say, first component commit or something along those lines?

**Review by kevinrodrigues — DISMISSED** [review]

**Review by kevinrodrigues — DISMISSED** [review]

**Comment by matthewhardern** [comment]
can you add it to the CI cache or do that in a follow up PR

File: packages/services/f-compatibility/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@justeat/f-compatibility",
+  "description": "Fozzie Compatibility - Service to check browser compatibilities and warn users with incompatible browsers",
+  "version": "0.1.0",
+  "maxBundleSize": "100kB",
+  "main": "dist/f-compatibility.umd.min.js", 
+  "source": "src/index.js",
```
Comment (DamianMullins) [inline packages/services/f-compatibility/package.json:8]: Can you add a `"module"` property here for ESM builds, please? i.e. `"module": "dist/f-compatibility.es.js",`

File: packages/services/f-compatibility/package.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@justeat/f-compatibility",
+  "description": "Fozzie Compatibility - Service to check browser compatibilities and warn users with incompatible browsers",
+  "version": "0.1.0",
+  "maxBundleSize": "100kB",
+  "main": "dist/f-compatibility.umd.min.js",
```
Comment (DamianMullins) [inline packages/services/f-compatibility/package.json]: I don't think Vite outputs a minified `*.umd.min.js` bundle by default — only a `*.umd.js` bundle.

File: packages/services/f-compatibility/package.json (hunk 2 above)
Comment (simonsandersje) [reply packages/services/f-compatibility/package.json:8]: Updated

File: packages/services/f-compatibility/package.json (hunk 3 above)
Comment (simonsandersje) [reply packages/services/f-compatibility/package.json]: Updated

**Review by matthewhardern — DISMISSED** [review]

**Review by DamianMullins — DISMISSED** [review]

**Review by dandel10n — APPROVED** [review]

**Review by FayeCarter — APPROVED** [review]

**Pull request merged**